frontend/src/components/welcome.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from 'react';
+import Script from 'react-load-script';
+import { Form, Button, Jumbotron } from 'react-bootstrap'
+
+class Search extends Component {
+
+  // Define Constructor
+  constructor(props) {
+    super(props);
+    this.handleScriptLoad = this.handleScriptLoad.bind(this);
+
+    // Declare State
+    this.state = {
+      city: '',
+      query: ''
+    };
+
+  }
+
+  render() {
+    return ( 
+      <div>
+        <Script url = "https://maps.googleapis.com/maps/api/js?key=KEY&libraries=places" onLoad = {this.handleScriptLoad}/> 
```

Comment:
i think you can style it like this  in order to be consistent with other files.
<Script 
    url = "https://maps.googleapis.com/maps/api/js?key=KEY&libraries=places" 
      onLoad = {this.handleScriptLoad}/>